overlord/managers_test.go

```diff
@@ -78,6 +78,8 @@ type mgrsSuite struct {
 	serveIDtoName map[string]string
 	serveSnapPath map[string]string
 	serveRevision map[string]string
+	serveOldPaths map[string][]string
```

Comment:
most of the changes here (other than the new test itself) are about making the mock store be epoch-aware. They stand on their own, but would be untested without the test, which would fail without the code in this PR... :man_shrugging: